eureka/src/main/java/com/linecorp/armeria/internal/common/eureka/InstanceInfo.java

```diff
@@ -36,7 +36,8 @@
 /**
  * An instance information.
  */
-@JsonIgnoreProperties(value = { "healthCheckUrlPath" }, ignoreUnknown = true)
+@JsonIgnoreProperties(
+        value = { "homePageUrlPath", "statusPageUrlPath", "healthCheckUrlPath" }, ignoreUnknown = true)
```

Comment:
Optional) `@JsonIgnore` would be more maintainable to exclude some fields from serialization.

Reply:
Oh, I forgot about it. 😉